worker/restore_map.go

```diff
@@ -285,6 +285,8 @@ func (m *mapper) processReqCh(ctx context.Context) error {
 	maxUid := uint64(0)
 
 	toBuffer := func(kv *bpb.KV, version uint64) error {
+		// Reset the StreamId to prevent ordering issues.
+		kv.StreamId = 0
```

Comment:
This will slow down the restore. Let's not do this.

Reply:
Yeah, it might slow down. I have assigned `math.MaxUint32` instead as this would not be used by any stream. This also saves us from some complexity like bulk loader.

Reply:
We are already reseting it for the data. Made the change for resetting it for schema.